deeplinkdispatch/src/main/java/com/airbnb/deeplinkdispatch/BaseDeepLinkDelegate.kt

```diff
@@ -583,6 +583,21 @@ open class BaseDeepLinkDelegate @JvmOverloads constructor(
         registries.flatMap { it.getAllEntries() }
     }
 
+    /**
+     * Get a map of all DeepLinkEntries and its duplicates, DeepLinkEntry objects that
+     * might be slightly different but will map to the same URL during app operation.
+     */
+    val duplicatedDeepLinkEntries: Map<DeepLinkEntry, List<DeepLinkEntry>> by lazy {
+        allDeepLinkEntries.mapNotNull { deepLinkEntry ->
```

Comment:
it may be nice to extract this function to be top level and take a list of deeplink entry and return the duplicates map.

In our test we have the set of registries, not this delegate